packages/dbml-core/src/export/DbmlExporter.js

```diff
@@ -263,59 +266,68 @@ class DbmlExporter {
     return strArr.length ? strArr.join('\n') : '';
   }
 
+  static getTableGroupSettings (tableGroup) {
+    const settings = [];
+
+    if (tableGroup.color) settings.push(`color: ${tableGroup.color}`);
+
+    return settings.length ? ` [${settings.join(', ')}]` : '';
+  }
+
   static exportTableGroups (tableGroupIds, model) {
     const tableGroupStrs = tableGroupIds.map(groupId => {
       const group = model.tableGroups[groupId];
       const groupSchema = model.schemas[group.schemaId];
+      const groupSettingStr = DbmlExporter.getTableGroupSettings(group);
 
-      return `TableGroup ${shouldPrintSchema(groupSchema, model)
-        ? `"${groupSchema.name}".` : ''}"${group.name}" {\n${
-        group.tableIds.map(tableId => {
+      const groupNote = group.note ? `  Note: ${DbmlExporter.escapeNote(group.note)}\n`: '';
+      const groupSchemaName = shouldPrintSchema(groupSchema, model) ? `"${groupSchema.name}".` : '';
+      const groupName = `${groupSchemaName}"${group.name}"`;
+
+      const tableNames = group.tableIds
+        .reduce((result, tableId) => {
           const table = model.tables[tableId];
           const tableSchema = model.schemas[table.schemaId];
-          return `  ${shouldPrintSchema(tableSchema, model)
-            ? `"${tableSchema.name}".` : ''}"${table.name}"`;
-        }).join('\n')}\n}\n`;
+          const tableName = `  ${shouldPrintSchema(tableSchema, model) ? `"${tableSchema.name}".` : ''}"${table.name}"`;
+          return result + `${tableName}\n`;
+        }, '');
+
+      return `TableGroup ${groupName}${groupSettingStr} {\n${tableNames}${groupNote}}\n`;
     });
 
     return tableGroupStrs.length ? tableGroupStrs.join('\n') : '';
   }
 
+  static exportStickyNotes (model) {
+    return reduce(model.notes, (result, note) => {
+      const escapedContent = `  ${DbmlExporter.escapeNote(note.content)}`;
+      const stickyNote = `Note ${note.name} {\n${escapedContent}\n}\n`;
+
+      // Add a blank line between note elements
+      if (result) return result + '\n' + stickyNote;
+      return result + stickyNote;
```

Comment:
- Can we just `return stickyNote` in the line 308?
- How about merging them into 1 line:
  ```js
  return result ? result + '\n' + stickyNote : stickyNote;
  ```

Reply:
Thanks for the suggestion a, I updated the code.